tests/unit/components/cylc/tree/tree.vue.spec.js

```diff
@@ -258,18 +276,15 @@ describe('Tree component', () => {
         })
         // the task proxy in our test data is "foo", so "bar" filter should have removed it
         wrapper.vm.filterTasks()
-        expect(wrapper.vm.$children.length).to.equal(4)
-        const taskProxy = wrapper.vm.$children.find((child) => child.$options._componentTag === 'tree-item')
-        expect(taskProxy).to.not.equal(null)
-        // cycle point is displayed (list.filter(() => return filtered=true means OK, filtered=false means remove me)
-        expect(taskProxy.filtered).to.equal(false)
+        // 3, 1 cycle point, 1 task proxy, and 1 job
+        const treeItems = wrapper.findAllComponents(TreeItem)
+        expect(treeItems.length).to.equal(3)
+        const taskProxy = treeItems.at(1)
         // task proxy is displayed
-        expect(taskProxy.$children[0].filtered).to.equal(false)
+        expect(taskProxy.vm.filtered).to.equal(false)
         wrapper.vm.removeAllFilters()
-        // cycle point is displayed (list.filter(() => return filtered=true means OK, filtered=false means remove me)
-        expect(taskProxy.filtered).to.equal(true)
         // task proxy is displayed
-        expect(taskProxy.$children[0].filtered).to.equal(true)
+        expect(taskProxy.vm.filtered).to.equal(true)
```

Comment:
These tests were broken!!!

I thought `children` would return the children components of `Tree`. And it does, but I thought the children were `TreeItem`. Turns out it brings several components, like the `TreeItem`s, but also all the Vuetify components (like `VTextField` and whatnot).

So fixed it by using the `Wrapper` and `WrapperArray` objects functions (and with some debugging :grimacing: ).

They got broken in this PR as I added new children components for the plus and minus icons.